cmd/shield/main.go

```diff
@@ -1485,11 +1486,23 @@ func main() {
 				return err
 			}
 
-			if err := RunJob(id, string(b)); err != nil {
+			var taskUUID string = ""
+			if taskUUID, err = RunJob(id, string(b)); err != nil {
```

Comment:
minor nit, but here I think I would stop using the `if <initializer>; <conditional> { ... }` construct and just assign `taskUUID, err := ...`